themes/acquia_claro/scss/acquia_cms_toolbar.scss

```diff
@@ -865,6 +865,12 @@
 
 .toolbar {
   .toolbar-tray-vertical {
+    .level-2 {
+      a {
+        color: $df-light-grey !important;
```

Comment:
Can we avoid !important with css specificity?

Reply:
Hey @ganesh41 , tried checking for changes after removing `!important` property. Changes are not reflected if we remove the property also I see !important applied to level-3 and level-4 menu items as well.